src/i18n.js

```diff
@@ -136,7 +136,6 @@ for (const locale of locales.keys()) {
 // EXPORT
 
 export default () => createI18n({
-  allowComposition: true,
```

Comment:
I think this is the default now? The release notes for Vue I18n v10 says that this option is no longer supported.